spec/relativetimeformat.html

```diff
@@ -21,12 +21,16 @@ <h1>InitializeRelativeTimeFormat ( _relativeTimeFormat_, _locales_, _options_ )<
           1. Let _options_ be ? ToObject(_options_).
         1. Let _opt_ be a new Record.
         1. Let _matcher_ be ? GetOption(_options_, `"localeMatcher"`, `"string"`, «`"lookup"`, `"best fit"`», `"best fit"`).
+        1. Let _numberingSystem_ be ? GetOption(_options_, `"numberingSystem"`, `"string"`, *undefined*, *undefined*).
```

Comment:
I think you should keep these new lines below the line 
```
Set _opt_.[[LocaleMatcher]] to _matcher_.
```

Reply:
Reordered; thanks.